e2e/pages/adf/cardViewComponentPage.ts

```diff
@@ -193,4 +195,40 @@ export class CardViewComponentPage {
         });
     }
 
+    async clearDateField() {
```

Comment:
a method called clearDateField should do just that, clear a field, not return a value.
you can have a separate method for getting the field value, and use that one in the expect in the test.
`expect(await cardViewPageComponent.clearDateField()).toBe('')` makes no sense at a first glance, cause you wouldn't expect clearDateField() to return a value